src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy

```diff
@@ -662,6 +664,119 @@ class LinkSpec extends HealthCheckSpecification {
         getIsl().srcSwitch.dpId | getIsl().srcPort | getIsl().dstSwitch.dpId | null      | "dst_port"
     }
 
+    @Tidy
+    def "Unable to delete inactive link with flowPath"() {
+        given: "An inactive link with flow on it"
+        def switchPair = topologyHelper.getNeighboringSwitchPair()
+        def flow = flowHelperV2.randomFlow(switchPair)
+        flowHelperV2.addFlow(flow)
+        def flowPath = pathHelper.convert(northbound.getFlowPath(flow.flowId))
+
+        // "Make all alternative paths unavailable (bring ports down on the source switch)"
```

Comment:
just make a pinned flow instead?

Reply:
I would like to leave it as is.
it should allow us to notice once system behavior is changed

Reply:
good idea, fixed